### PR TITLE
Merge xcode 9.3 into d15-7

### PR DESCRIPTION
**Pull request by VincentDondain**

### CODE REVIEW

**Comment by VincentDondain** [comment]
_Note: xtro might fail (will look at it later when I go home and after I rebuild)._

File: src/SceneKit/SCNCompat.cs [hunk 1]
```diff
@@ -97,6 +97,7 @@ public static SCNAnimationEvent Create (nfloat keyTime, SCNAnimationEventHandler
 		}
 	}
 #endif
+<<<<<<< HEAD
```
Comment (dalexsoto) [inline src/SceneKit/SCNCompat.cs]: Diff noise

File: src/SceneKit/SCNCompat.cs [hunk 2]
```diff
@@ -111,4 +112,6 @@ static public void AddAnimation (this ISCNAnimatable self, SCNAnimation animatio
 		}
 	}
 #endif
+=======
```
Comment (dalexsoto) [inline src/SceneKit/SCNCompat.cs]: Diff noise

File: src/StoreKit/StoreProductParameters.cs [hunk 3]
```diff
@@ -36,19 +36,9 @@
...
-		public StoreProductParameters ()
-			: base (new NSMutableDictionary ())
-		{
-		}
-
-		public StoreProductParameters (NSDictionary dictionary)
-			: base (dictionary)
-		{
-		}
-
```
Comment (spouliot) [inline src/StoreKit/StoreProductParameters.cs]: needs double-checking api-diff

**Comment by chamons** [comment]
Yes, I've almost hit squash more than one time in doing merges. Talk about a nightmare.

**Comment by VincentDondain** [comment]
The API diff looks reasonable to me: https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/6825/API_diff_(from_stable)/. @chamons ?

**Pull request merged**